io.github.ferraridamiano.ConverterNOW.appdata.xml

```diff
@@ -0,0 +1,53 @@
+    </p>
+    <p>
+      Check the repository on GitHub: https://github.com/ferraridamiano/ConverterNOW
+    </p>
+    <p>
+      We also have a Web App! 🖥️ Check it out here: https://ferraridamiano.github.io/ConverterNOW/
+    </p>
+    <p>
+      🌍 Converter NOW is free, without ads, it does not require any permission and we don&apos;t collect your data. If you believe in this project and want to support the developer make a donation: https://www.paypal.me/DemApps 🌍 Thanks to you we can always stay on 🔝
+    </p>
+    <p>
+      Converter NOW can convert 200+ units of mesurement and 30+ currencies 🎉🎉
+    </p>
+    <p>
+      Here are some of the physical quantity that Converter NOW is able to convert: - Currencies: Dollars, Euro, Pound, Rupee, Yen, etc - Length: meters, inches, miles, yards, light years, etc. - Area: square meters, hectares, acres, etc. - Volume: cubic meters, liters, gallons, pints, spoons, etc. - Time: seconds, hours, days, years, weeks, millennia, etc. - Temperature: Centigrade, Fahreneit, Kelvin - Speed: meters per second, kilometers per hour, knots, etc. - Mass: grams, pounds, tons, atomic mass units, etc. - Force: Newton, dyne, pound-force, poundal, etc - Pressure: pascal, bar, atomsfere, psi, etc. - Energy: Joule, calories, kilowatt hours, etc. - Power: Watt, kilowatt, horse power, etc. - Fuel consumption: Miles per Gallon, Kilometres per liter, etc. - Numeral systems: decimal, binary, hexadecimal etc. - Torque: Newton metre, pound-force foot, poundal metre, etc. - Digital data: Nibble, bit, byte, kibibit, kibibyte, etc. - Shoes size: UK, India, Europe, USA, Japan, etc. - Angles: degree, radians, minutes, etc. - SI prefix: kilo, mega, giga, tera, milli, micro, nano, etc.
+    </p>
+  </description>
+
+  <launchable type="desktop-id">io.github.ferraridamiano.ConverterNOW.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/ferraridamiano/ConverterNOW/master/fastlane/metadata/android/en-US/images/phoneScreenshots/1.jpeg</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/ferraridamiano/ConverterNOW/master/fastlane/metadata/android/en-US/images/phoneScreenshots/2.jpeg</image>
+    </screenshot>
+    <screenshot>
+      <image>https://raw.githubusercontent.com/ferraridamiano/ConverterNOW/master/fastlane/metadata/android/en-US/images/phoneScreenshots/3.jpeg</image>
+    </screenshot>
+  </screenshots>
```

Comment:
`content_rating` and `releases` are missing.

Also has this been submitted for inclusion upstream?

Reply:
I'm sorry, but I don't find any documentation about what thos tags means. Could you please link it? Thank you!

Didn't know I should also upload the screenshot to this project. I will add them

Reply:
I never said that.

Reply:
I've just added `content_rating` and `releases`. Could you please elaborate on "Also has this been submitted for inclusion upstream?". I don't understand what does it means

Reply:
that the file should be managed by the upstream project (for future versions), It is not specific to flatpak.